lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -19,8 +18,8 @@
     </form>
 
     <a href="" class="action-primary action-create">
-        <i class="icon fa fa-plus"></i>
-        <%- gettext('Add Cohort') %>
+        <i class="icon fa fa-plus" aria-hidden="true"></i>
+        <%- gettext('Add Cohort Group') %>
```

Comment:
Might've been something during the rebase. I erred on the side of "more descriptive". I'll remove.